Quantum-Control-Applications/Optically addressable spin qubits/NV center in a confocal setup/09c_xy8_order.py

```diff
@@ -43,7 +43,7 @@
 # Must be a multiple of 2 clock cycles to ensure that tau_half is a multiple of a single clock cycle
 tau = 2 * 50
 order_vec = np.arange(1, 21, 1, dtype=int)  # order vector for varying the order n of the XY8-n measurement
-n_avg = 1_000
+n_avg = 1_000_000
```

Comment:
Is 1000000 averages really needed as the default value?
I usually prefer to have a smaller number of averages to ensure that the data is correct quickly, before increasing it to improve the SNR, but I leave it to you.